src/components/WorkSection.js

```diff
@@ -220,10 +219,9 @@ class WorkSection extends PureComponent {
               </h2>
               {get(activeProject, 'fields.caseStudySlug', '') && (
                 <Link
-                  className="small link text-no-decoration pb1 pr2"
```

Comment:
can u snip the CSS class you made for this too?

Reply:
Just saw there is a `decoration-none` class that was already there so I'm using that instead now!